Gems/Atom/Component/DebugCamera/Code/Source/CameraComponent.cpp

```diff
@@ -272,7 +272,10 @@ namespace AZ
             else if (m_componentConfig.m_target)
             {
                 const auto& viewport = m_componentConfig.m_target->GetViewport();
-                m_aspectRatio = viewport.m_maxX / viewport.m_maxY;
+                if (viewport.m_maxX > 0.0f && viewport.m_maxY > 0.0f)
+                {
+                    m_aspectRatio = viewport.m_maxX / viewport.m_maxY;
+                }
```

Comment:
I realize this was pre-existing, but does the viewport always have 0 for the minX & minY?

Reply:
It's presently always the case in all case I'm aware of, though I agree this logic should probably take the minimum viewport extents into account. If possible I'd prefer to address that case in the future, as I don't have a good test case to validate against and I'd like to keep this change minimal to unblock our automation.